settings.php

```diff
@@ -39,10 +39,10 @@
             get_string('setting:notificationtype_desc', 'local_edit_course_notification'),
             get_string('notificationtype', 'local_edit_course_notification'),
```

Comment:
this should be \core\output\notification::NOTIFY_INFO

Reply:
Thanks for finding the bug.
I updated the code.